### ISSUE TITLE
Support proxy authentication (other than basic)

### ISSUE THREAD

**Issue by thekvs** [httpie/cli#221]
As far as I can see httpie doesn't support authentication on proxies. It would be nice to have such feature.

**Comment by prp20** [comment]
thanks a million it just worked fine and awesome.

**Comment by Almad** [comment]
Digest / NTLM authentication is blocked by urllib3: https://github.com/urllib3/urllib3/issues/242
